index.html

```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <meta charset="UTF-8" />
+    <!-- Font Awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+    />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <!-- Nav Section -->
+        <nav class="menu">
+          <a href="#" class="logo"><h2>My Logo</h2></a>
+
+          <ul>
+            <li>
+              <a href="#" class="active" onclick="markMenuItem(this)"
+                >Portfolio</a
+              >
+            </li>
+            <li><a href="#" onclick="markMenuItem(this)">About</a></li>
+            <li><a href="#" onclick="markMenuItem(this)">Contact</a></li>
+          </ul>
+
+          <div id="menuBtn" class="active">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Headline Section -->
+      <main>
+        <section class="info">
+          <h1>I’m Mariane Glad to see you!</h1>
+          <p id="infos">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </p>
+          <p id="infoss">LET’S CONNECT</p>
+          <ul class="socialMedia">
+            <li>
+              <a href=""> <img src="assets/twitter.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="assets/linkedin.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="assets/mail.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="assets/github.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""> <img src="assets/fingers.svg" alt="" /></a>
+            </li>
+          </ul>
+        </section>
+      </main>
+
+      <!-- Work Section -->
+      <article>
+        <ul id="work">
+          <li class="work" id="work-img-1" >
```

Comment:
- [ ] Excelent work creating the cards. Kindly follow the best semantic tags in HTML. Change the `article` for a `section` and `ul` for an `article`. And inside the `article` tag you should have the information of each card. It's not necessary to wrap a card inside a `ul` or `li` since they are not items they are a conjunct of items, this is why you need to create an article for each card instead of a `ul` or `li` 🥇🥇